src/pages/GE_TokenizeCard.page

```diff
@@ -69,10 +69,14 @@
             if (event && event.origin !== lightningOrigin) {
                 // Reject any messages from an unexpected origin
                 return;
+
             } else if (event.data && event.data.action === 'createToken') {
                 if (event.data.nameOnCard) {
                     nameOnCard = event.data.nameOnCard;
+                } else {
+                    nameOnCard = 'Not provided';//default
```

Comment:
The name needs to be reset every time the card tokenization is required. See AC3 in the WI for the scenario.